.github/workflows/update-pinned-reqs.yml

```diff
@@ -34,14 +34,14 @@ jobs:
     # action.
 
     - name: Generate token
-      uses: tibdex/github-app-token@v2
+      uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
```

Comment:
It seems reasonable to pin actions by the commit hash (for extra security) if they are released by individuals or small organizations.